### PR TITLE
Add flask-caching

### PR DESCRIPTION
**Pull request by halldc**
Created using PyPI skeleton.

I am using `flask-caching v1.3.3` as the initial conda package version, because this is required by `airflow v1.10.0`. Once this is built, we can update to `flask-caching v1.4.0`.

### CODE REVIEW

**Comment by moorepants** [comment]
I think this should be a noarch package: https://conda-forge.org/docs/meta.html#building-noarch-packages

File: recipes/flask-caching/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5af1759e5ae3424abec918537f0201a1476ae9442452bcb5c8787468a9de0f5a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
```
Comment (marcelotrevisani) [inline recipes/flask-caching/meta.yaml]: As pointed by @moorepants this package may be ``noarch: python``

**Comment by halldc** [comment]
@moorepants @marcelotrevisani Thanks for the tip! I've made the change.

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks to put this in ``conda-forge``! :+1:

**Pull request merged**